packages/jest-transform/src/ScriptTransformer.ts

```diff
@@ -607,13 +617,16 @@ const wrap = (content: string, ...extras: Array<string>) => {
     ...extras,
   ]);
 
-  return (
+  const wrapper = `function(${Array.from(globals).join(',')}){`;
```

Comment:
Looking at https://github.com/istanbuljs/v8-to-istanbul/pull/40, this might not be correct. @bcoe could you advise on the `wrapperLength` argument? 🙂

Reply:
the wrapper length was addressing a historical issue with Node.js, in which it inserted a wrapper around any JavaScript before executing it.

Newer versions of Node.js use a new API, such that the first byte offset is 0 (rather than the wrapper length) ... so, Ideally we shouldn't need this value any longer in most contexts.

Reply:
What new API is that? We still have a wrapper. We've never had issues with it in Jest though, so maybe we don't need to pass anything?

Reply:
the pertinent change was this PR:

https://github.com/nodejs/node/pull/21573

Basically, if you were executing a script with Node.js, there was a wrapper that inserted the globals:

```js
function (fs, require, etc) {

}
```

The first `N` bytes would then effect the start position of your script, if tracking coverage. tldr; check the `html` reports, they're the best way to see if there are any offset issues. Mising coverage should have logical starting and stopping positions if the wrapper is correct.